app/models/option_sets/import.rb

```diff
@@ -29,24 +35,73 @@ def cleaner
     end
 
     def create_option_set_and_copy_any_errors(headers, geographic)
-      self.option_set = OptionSet.create(
+      params = {
         mission: mission,
         name: name,
-        levels: headers.map { |h| OptionLevel.new(name: h) },
         geographic: geographic,
         allow_coordinates: geographic,
         root_node: OptionNode.new
-      )
+      }
+
+      params[:levels] = headers.map { |h| OptionLevel.new(name: h) } if headers.any?
+
+      self.option_set = OptionSet.create(params)
       copy_validation_errors_for_row(1, option_set.errors) unless option_set.valid?
     end
 
-    def process_row(row, row_idx)
+    def process_row(row, row_idx, languages)
       # Metadata about the option is included at the end of the row array.
       metadata = row.extract_options!
+      if languages.any?
+        translation_row(row, row_idx, metadata, languages)
+      else
+        multilevel_row(row, row_idx, metadata)
```

Comment:
This is not necessarily multilevel, just untranslated, so I will rename this for clarity.